easybuild/easyconfigs/g/googledrivedownloader/googledrivedownloader-0.4-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'googledrivedownloader'
+version = '0.4'
+
+homepage = 'https://github.com/ndrplz/google-drive-downloader'
+description = """Minimal class to download shared files from Google Drive."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['4b34c1337b2ff3bf2bd7581818efbdcaea7d50ffd484ccf80809688f5ca0e204']
+
+builddependencies = [('binutils', '2.31.1')]
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+download_dep_fail = True
+use_pip = True
+
+options = {'modulename': 'google_drive_downloader'}
```

Comment:
You should be able to remove this line if you go with `google-drive-downloader` as `name`